WordPress/Classes/ViewRelated/Stats/Referrer Details/ReferrerDetailsViewModel.swift

```diff
@@ -20,6 +21,10 @@ extension ReferrerDetailsViewModel {
     func update(with data: StatsTotalRowData) {
         self.data = data
     }
+
+    func setLoadingState(_ value: Bool) {
```

Comment:
Question: Is there a benefit to exposing just the setter vs making the entire var public?

Reply:
I think it's easier to track what's going on that way, it's more explicit. But really I don't have any strong argument against the other approach. Should I leave it as is, or make the property non-private?

Reply:
Ah makes sense. I think we can leave it the way it is right now. I don't have a strong opinion either way.